STNetwork/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/network/exceptions/ContainerErrorsException.kt

```diff
@@ -82,11 +82,21 @@ sealed class ContainerErrorsException(
     class TooManyCodesGeneratedException(requestContextId: String) :
         ContainerErrorsException(429, "Too many codes generated", requestContextId)
 
+    /**
+     * Exception indicating that the daily quota have been exceeded.
+     * This corresponds to an HTTP 429 Too Many Requests status.
+     *
+     * @param requestContextId The request context id send by the backend to track the call
+     */
+    class DailyQuotaExceededException(requestContextId: String) :
+        ContainerErrorsException(429, "Daily transfer limit reached", requestContextId)
```

Comment:
Technically, you could reuse the `DAILY_TRANSFER_LIMIT` constant you declared below.